web/package.json

```diff
@@ -37,16 +40,10 @@
         "tsx": "^4.16.5",
         "typescript": "^5.5.4",
         "typescript-eslint": "^8.0.0",
-        "wdio-chromedriver-service": "^8.1.1",
-        "wdio-edgedriver-service": "^3.0.3",
-        "wdio-geckodriver-service": "^5.0.2",
-        "webdriverio": "^8.33.1",
+        "webdriverio": "^8.40.0",
         "webpack": "^5.93.0",
         "webpack-cli": "^5.1.4"
     },
-    "optionalDependencies": {
-        "chromedriver": "^127.0.1"
```

Comment:
This was optional before - should it stay that way...?

Reply:
Maybe. `wdio-chromedriver-service`, `wdio-edgedriver-service`, and `wdio-geckodriver-service` were devDependencies so I wasn't sure which was more correct.